DependencyInjection/DoctrineMongoDBExtension.php

```diff
@@ -218,6 +213,8 @@ protected function loadDocumentManagers(array $dmConfigs, $defaultDM, $defaultDB
      * @param string           $defaultDM       The default document manager name
      * @param string           $defaultDB       The default db name
      * @param ContainerBuilder $container       A ContainerBuilder instance
+     *
+     * @return void
      */
     protected function loadDocumentManager(array $documentManager, $defaultDM, $defaultDB, ContainerBuilder $container)
```

Comment:
```suggestion
     */
    protected function loadDocumentManager(array $documentManager, $defaultDM, $defaultDB, ContainerBuilder $container): void
```

It would be nice to revisit later if `protected` makes sense.